datadog/resource_datadog_synthetics_test_.go

```diff
@@ -523,6 +535,13 @@ func buildSyntheticsTestStruct(d *schema.ResourceData) *datadogV1.SyntheticsTest
 		allowInsecure, _ := strconv.ParseBool(attr.(string))
 		options.SetAllowInsecure(allowInsecure)
 	}
+	if attr, ok := d.GetOk("options.retry"); ok {
+		retryCount, _ := strconv.Atoi(attr.(string))
+		retry := datadogV1.SyntheticsTestOptionsRetry{}
+		retry.SetCount(int64(retryCount))
+		retry.SetInterval(300)
```

Comment:
Should we add support for retry interval right now? It's going to be awkward to add it later on if retry is a int instead of a struct.

Reply:
You're right, I'll change that to a struct 👍